apps/web/src/pages/integrations/components/NovuInAppRemoveBranding.tsx

```diff
@@ -33,47 +22,73 @@ const SwitchLabel = styled.div`
   line-height: 17px;
 `;
 
+const PopoverContent = () => {
+  const navigate = useNavigate();
+
+  return (
+    <Group spacing={8}>
+      <Text>Upgrade your billing plan to remove Novu branding</Text>
+      <Button
+        size="xs"
+        variant="light"
+        onClick={() => {
+          navigate(ROUTES.MANAGE_ACCOUNT_BILLING);
+        }}
+      >
+        View plans
+      </Button>
+    </Group>
+  );
+};
+
 export const NovuInAppRemoveBranding = ({ control }: { control: any }) => {
   const { apiServiceLevel } = useSubscription();
+  const { colorScheme } = useColorScheme();
   const isImprovedBillingEnabled = useFeatureFlag(FeatureFlagsKeysEnum.IS_IMPROVED_BILLING_ENABLED);
 
-  if (!isImprovedBillingEnabled) {
+  if (!isImprovedBillingEnabled && !IS_EE_AUTH_ENABLED) {
```

Comment:
`IS_EE_AUTH_ENABLED` now when Clerk == EE, we need to retire this variable or rename to e.g. `NOVU_ENTERPRISE`

Created separate task for this